src/components/EmptyResults.vue

```diff
@@ -0,0 +1,27 @@
+<template>
```

Comment:
if you component doesn't do anything please use functional component
https://vuejs.org/v2/guide/render-function.html#Passing-Attributes-and-Events-to-Child-Elements-Components
https://www.youtube.com/watch?v=7lpemgMhi0k